test/algorithms/set_operations/intersection/intersection.cpp

```diff
@@ -314,11 +312,10 @@ void test_areal()
 #endif
 
     TEST_INTERSECTION(issue_1229, 0, -1, 0);
-    
+
     {
-        // For CCW this test case currently reports as invalid
         ut_settings settings;
-        settings.set_test_validity(! is_ccw);
+        settings.set_test_validity(true);
```

Comment:
This works - but you can also remove the settings and just call `TEST_INTERSECTION` (here and below)